pkg/codegen/java/gen.go

```diff
@@ -507,6 +507,11 @@ func (pt *plainType) genInputType(ctx *classFileContext) error {
 			false,               // outer optional
 			false,               // inputless overload
 		)
+
+		if isOutput, _ := propTypes[i].UnOutput(); !isOutput && propertiesAsOutputs {
```

Comment:
this code will be used in only one place and is not universal, moreover it is similar to code already in typeString

Reply:
as noted above, since I don't have an idea how to address my concern, feel free to ignore this comment, sorry about that